tests/spread/integration/libarchive-tools/task.yaml

```
@@ -0,0 +1,26 @@
+summary: Integration tests for libarchive-tools
+
+execute: |
+  rootfs="$(install-slices libarchive-tools_bins)"
+  chroot "${rootfs}/" bsdcat --version
+  chroot "${rootfs}/" bsdcpio --version
+  chroot "${rootfs}/" bsdtar --version
+  chroot "${rootfs}/" bsdunzip --version
+  echo "Hello, World" > "${rootfs}/testfile.txt"
+  test "$(chroot "${rootfs}/" bsdcat testfile.txt)" = "Hello, World"
+  chroot "${rootfs}/" bsdtar -cf /testfile.tar /testfile.txt
+  if [ ! -f "${rootfs}/testfile.tar" ]; then 
+    echo "bsdtar failed to create archive: testfile.tar"
+    exit 1
+  fi
+  cp "${rootfs}/testfile.txt" "${rootfs}/testfile_orig.txt"
+  chroot "${rootfs}/" bsdtar -xf testfile.tar
+  test "$(cat ${rootfs}/testfile.txt)" = "$( cat ${rootfs}/testfile_tar.txt)"
```

```suggestion
  test "$(cat ${rootfs}/testfile.txt)" = "$( cat ${rootfs}/testfile_orig.txt)"
```
I believe you also want to replace this occurrence.